.readthedocs.yml

```diff
@@ -14,4 +14,5 @@ python:
   install:
     - method: pip
       path: .
-    - requirements: docs/requirements.txt
```

Comment:
Make sure you delete "docs/requirements.txt" and make sure it is not set that way in the Admin panel, just in case...

(Though not sure if that will fix your error or not)

Reply:
Ooooh, yes! The admin panel was not changed when I switched to `pyproject.toml`. Now I'm pointing to `docs/conf.py` and leaving blank the requirements file. Hopefully that will do the trick.